### ISSUE TITLE
Is it possible to return a variable then is it on the view?

### ISSUE THREAD

**Issue by ellemaker** [webwizo/laravel-shortcodes#50]
propose
    return "<?php $test = {$value}; ?>";

**Comment by webwizo** [comment]
I don't understand your question.

**Comment by ellemaker** [comment]
![Screenshot_63](https://user-images.githubusercontent.com/4677416/67584511-6bebc980-f780-11e9-82ed-13164e6f847c.png)
![Screenshot_64](https://user-images.githubusercontent.com/4677416/67584515-6c846000-f780-11e9-80e9-152a350b7992.png)
![Screenshot_65](https://user-images.githubusercontent.com/4677416/67584516-6c846000-f780-11e9-85fe-ba57744679f8.png)
![Screenshot_62](https://user-images.githubusercontent.com/4677416/67584518-6d1cf680-f780-11e9-8453-846374eb68e8.png)

see screenshot

**Comment by webwizo** [comment]
Let me see, what I can do.

**Comment by ellemaker** [comment]
so I want to use $variable in the view like {{ $variable }}

**Comment by webwizo** [comment]
So why need Shortcode? You can use create Blade Extended.

**Comment by ellemaker** [comment]
what I really want 

```
[webapp id='123123' collection='listofteams']
	@foreach ($listofteams as $item)
		<li>name</li>
	@endforeach
```

**Comment by webwizo** [comment]
Nice idea, dude. This feature can be added, but I don't think I can do it earlier :(

**Comment by ellemaker** [comment]
yes, I already made those things. but I want to change the layout in every collections. that's why we want it to foreach in the view.

**Comment by webwizo** [comment]
But you can add one more attribute like ```[webapp wrap="li" parent="ul"]``` this way you can create multiple layouts.

**Comment by ellemaker** [comment]
Thank you for helping me on this but I think it would be much easier if the we can use the variable that was return in shortcode or return a php tag(like what is in my screenshot).
